tests/test_execution.py

```diff
@@ -311,7 +260,8 @@ def test_aggregatetuple(global_execution_env):
 
 
 @pytest.mark.slow
-def test_aggregate_composite_traintuples(global_execution_env):
+def test_aggregate_composite_traintuples(factory, clients, default_datasets, default_objectives,
```

Comment:
I think it would be better to not mix all assets and assets from node.

So either use `default_datasets, default_objectives` or `default_dataset_1, default_dataset_2, default_objective_1, default_objective_2`.

Reply:
I agree, I'll change this